### PR TITLE
Signup page, Login page, Name input page, Gender input page, Interest Page Done

### PR DESCRIPTION
**Pull request by tanishqpati**
Pages Done in this PR:
- Signup Page 
- Login Page
- Name input Page
- Gender input Page
- Interest input Page
@imadityaanand helped me in this PR. Thank you Aditya : )

### CODE REVIEW

File: src/pages/GenderPage/GenderPage.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
+import React, { useState } from "react";
+import { TextField } from "@material-ui/core";
+import "./GenderPage.css";
+import { useNavigate } from "react-router-dom";
+
+const GenderPage = () => {
+  const navigate = useNavigate();
+  const [gender, setGender] = useState("");
+  const isFormValid = !!gender;
```
Comment (PatelNeelMahesh) [inline src/pages/GenderPage/GenderPage.js]: Here no need to use `!` twice. It's same as you not using it.
`!!gender` and `gender` will have same value.

File: src/pages/InterestPage/Interest.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
+import React, { useState } from "react";
+import { TextField } from "@material-ui/core";
+import "./Interest.css";
+import { useNavigate } from "react-router-dom";
+
+const Interest = () => {
+  const navigate = useNavigate();
+  const [gender, setGender] = useState("");
+  const isFormValid = !!gender;
```
Comment (PatelNeelMahesh) [inline src/pages/InterestPage/Interest.js]: Here no need to use `!` twice. It's same as you not using it.
`!!gender` and `gender` will have same value.

**Review by PatelNeelMahesh — APPROVED** [review]

File: src/pages/GenderPage/GenderPage.js (hunk 1 above)
Comment (tanishqpati) [reply src/pages/GenderPage/GenderPage.js]: Removed **!!** from gender page.

File: src/pages/InterestPage/Interest.js (hunk 2 above)
Comment (tanishqpati) [reply src/pages/InterestPage/Interest.js]: Removed **!!** from interest page.

**Pull request merged**